generators/csharp/codegen/src/asIs/RawGrpcClient.Template.cs

```diff
@@ -47,7 +47,7 @@ public CallOptions CreateCallOptions(
         CancellationToken cancellationToken = default
     )
     {
-        var metadata = new Metadata();
+        var metadata = new global::Grpc.Core.Metadata();
```

Comment:
what's this part? `global::Grpc.Core.`

Reply:
This helps guard against the edge case where users specify their own `Metadata` type (which will be fairly common). We don't use the global scope for the other gRPC utilities since it's far less likely that they conflict (e.g. `GrpcChannelOptions`).

We normally handle this well when we generate the files dynamically, but we don't have that auto-resolution given that this is an as-is file.